libgadget/blackhole.c

```diff
@@ -853,13 +1247,24 @@ blackhole_accretion_reduce(int place, TreeWalkResultBHAccretion * remote, enum T
             BHP(place).MinPotVel[k] = remote->BH_MinPotVel[k];
         }
     }
+    
+    /****************************************************************************/
+    if(HaloMinPot[PI] > remote->BH_HaloMinPot)
```

Comment:
I think this is MinPotPos but over a wider area? Please make this some kind of debug code.